calcom/cal.com
# fix: Add signup disabled flag check on signup route

## What does this PR do?



(Just to note that in the below video I am refreshing the server again and again is just to refresh the cache, so that I don't have to wait for 5 mins.)

https://www.loom.com/share/5246b4b7e13b4b759086f74d57fa19d5?sid=9eadf523-4c58-440b-bdc6-31a96cee0152

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Create a POST request to signup API as demonstrated in the video. This should be successful.
- Then disable the signup from the admin settings.
- Wait for 5 mins. (cache expiration time)
- Then again make the above request. This should now say that signup is disabled.
